apps/web/pages/event-types/index.tsx

```diff
@@ -396,7 +424,12 @@ export const EventTypeList = ({
           const isChildrenManagedEventType =
             type.metadata?.managedEventConfig !== undefined && type.schedulingType !== SchedulingType.MANAGED;
           return (
-            <li key={type.id}>
+            <li
+              key={type.id}
+              draggable={!readOnly}
+              onDragStart={(e) => dragStart(e, index)}
+              onDragEnter={() => dragOver(index)}
+              onDragEnd={drop}>
```

Comment:
i wonder if we put this on the up and down buttons:

![CleanShot 2024-01-07 at 10 39 28@2x](https://github.com/calcom/cal.com/assets/8019099/432af081-95f6-4c29-875c-4cecf4117ca5)


and give it class="pointer-grab"

Reply:
Ok I will add